MaterialMonitor/Properties/MaterialMonitor.resx

```diff
@@ -141,6 +159,12 @@
   <data name="p2" xml:space="preserve">
     <value>Note that if your inventory is incorrect you will need to start or restart Elite and this information will be updated.</value>
   </data>
+  <data name="p3" xml:space="preserve">
+    <value>Note: Restrictive station distance setting may significanlty increase search time</value>
```

Comment:
Typo: `significantly`
Proposed revision: `Tip: Maximize the allowed station distance to minimize search times.`

Reply:
Done.